src/components/modals/send-modal/index.js

```diff
@@ -74,12 +108,13 @@ const SendModal = ({ show, handleModal, utxo, onSale }) => {
 
     const feeRateOnChange = (evt) => setSendFeeRate(evt.target.value);
 
+    const boostRequired = !!utxo && !!sendFeeRate && outputValue(utxo, sendFeeRate) < MIN_OUTPUT_VALUE;
+
     const submit = async () => {
         setIsSending(true);
-        await sendUtxo().catch((err) => {
+        await sendUtxo(boostRequired).catch((err) => {
             console.error(err);
             alert(err);
-            return false;
```

Comment:
nit: Why removal of true/false returns?n

Reply:
the linting forced me to be consistent, either always return something or never return something. we can put them all back and add ones for the new returns i added